### ISSUE TITLE
Some questions about loss function

### ISSUE THREAD

**Issue by zq-hznu** [D1noFuzi/cmd_styletransfer#1]
Hello, senior, I am a graduate student majoring in electronic information from Hangzhou Normal University, Hangzhou, China. I read your article "In the light of feature distributions: moment matching for Neural Style Transfer" and I am deeply inspired. I read your code, but I don’t quite understand the high-order moment loss function. I have some puzzles to ask: 1. This is a method based on image iteration, whether the input of ContentLoss is the feature map of the  content image and generated image extracted by relu4_1  , the input of StyleLoss is  the feature map of the style image and generated image extracted by multiple relus ? 2. Can you explain in detail how the style loss is calculated?
Scientific research is not easy, look forward to it!

**Comment by adaxidedakaonang** [comment]
正在啃作者的paper

**Comment by zq-hznu** [comment]
> 正在啃作者的paper

你好，请问我能请教你一些关于这篇文章损失函数的问题吗？

**Comment by adaxidedakaonang** [comment]
要问那个yita吗（公式10）

**Comment by zq-hznu** [comment]
> 要问那个yita吗（公式10）
这个是从内容、风格图片和风格化图片的特征图计算mse损失吗，高阶矩在公式中好像没有体现，代码中的k是否就是高阶矩系数？

**Comment by adaxidedakaonang** [comment]
作者利用CMD来作为风格loss，k是个超参数，是阶数

**Comment by zq-hznu** [comment]
> 作者利用CMD来作为风格loss，k是个超参数，是阶数

StyleLoss的输入是什么，target 和 x 分别代表什么？
这个函数的倒数第二行：loss = loss + self.weights[i - 1] * self.rmse(m_x, self.m_ys[i - 2])，好像两个矩阵不对应呀！

**Comment by adaxidedakaonang** [comment]
font{
        line-height: 1.6;
    }
    ul,ol{
        padding-left: 20px;
        list-style-position: inside;
    }


    
    
        
            
        
    
    
        似乎你的疑问是代码方面的， 代码我没有看，只看了论文
        
    
        
        

        
    

    On 9/29/2021 ***@***.***> wrote： 






作者利用CMD来作为风格loss，k是个超参数，是阶数



StyleLoss的输入是什么，target 和 x 分别代表什么？

这个函数的倒数第二行：loss = loss + self.weights[i - 1] * self.rmse(m_x, self.m_ys[i - 2])，好像两个矩阵不对应呀！
